### ISSUE TITLE
TS2353 false positive for class:!foo Svelte class directive

### ISSUE THREAD

**Issue by esafak** [harshmandan/svelte-check-native#14]
## Summary

`class:` directives that use Tailwind important utility names (e.g. `class:!font-semibold={cond}`) are accepted by the Svelte compiler, but `svelte-check-native` reports `TS2353` as if the directive were an unknown DOM prop.

## Version

- `svelte-check-native`: `0.7.2`
- `@typescript/native-preview`: `7.0.0-dev.20260427.1`
- Svelte project (Svelte 5)

## Repro

Minimal Svelte:

```svelte
<script lang="ts">
  let unread = true;
</script>

<span class:!font-semibold={unread}>Hello</span>
```

Run:

```bash
svelte-check-native --tsconfig ./tsconfig.json
```

## Actual

`TS2353` about unknown property on `<span>` (class directive appears to be treated like a DOM prop).

## Expected

No TS2353 here. This is valid Svelte class-directive syntax, and the Svelte compiler accepts it.

## Additional context

- Tailwind output for `!font-semibold` is emitted as expected (including `!important`).
- This appears to have surfaced after upgrading `svelte-check-native` from `0.7.1` to `0.7.2`.

If helpful, I can provide a minimal public repo reproducing this.
